frontend/app/routes/protected/person-case/@types.d.ts

```diff
@@ -55,7 +57,7 @@ declare module 'express-session' {
         province: string;
       };
       birthDetails:
-        | { country: 'CAN'; province: string; city: string; fromMultipleBirth: boolean }
+        | { country: serverEnvironment.PP_CANADA_COUNTRY_CODE; province: string; city: string; fromMultipleBirth: boolean }
```

Comment:
```suggestion
        | { country: ServerEnvironment['PP_CANADA_COUNTRY_CODE']; province: string; city: string; fromMultipleBirth: boolean }
```

Reply:
`serverEnvironment` is not type definition. You would have to use `(typeof serverEnvironment)['PP_CANADA_COUNTRY_CODE']` to make it works. If I save this file the `import type { serverEnvironment } from '~/.server/environment';` import is removed. Using `ServerEnvironment` type import works though. `serverEnvironment.PP_CANADA_COUNTRY_CODE` resolved to `any` at the moment.